modules.py

```diff
@@ -14,56 +12,39 @@ def obj_terms(x_in, x_name, p_in: dict):
 
     # run each module 
     wave_out = wave_climate(wec,wave_in)
-    fish_yield = fish_yield_func(wave_out,pen, fish)
+    fish_yield = fish_yield_func(wave_out, pen, fish)
     wec.P_gen = power(wec, wave_in)
     
     price = econ(wec, pen)
    
     # outputs
     cost_per_yield = price/fish_yield 
-    #print(x_in, price, fish_yield, cost_per_yield)
-    
-    return cost_per_yield, price, fish_yield
+    return cost_per_yield, price, fish_yield, pen.price, wec.price
 
 def ineq_constraint(x_in, x_name, p):
     # merge input dicts
     wec, wave_in, pen, fish = input_merge(x_in, x_name, p)
 
     # run each module 
     wave_out = wave_climate(wec,wave_in)
-    fish_yield = fish_yield_func(wave_out,pen, fish)
+    fish_yield = fish_yield_func(wave_out, pen, fish)
     wec.P_gen = power(wec, wave_in)
     carrying_capacity = carrying_capacity_func(pen, fish)
     
-    P_gen_cons = wec.annual_energy - pen.power
-    #print('P_gen_cons=', wec.annual_energy, pen.power, P_gen_cons)
-    #P_gen_cons = wec.P_gen - pen.power
-    
-    fish_yield_cons = (carrying_capacity - fish_yield)   # Multiply 0.001 to get a similar order with P_gen_cons
-    #fish_yield_cons = (carrying_capacity - fish_yield / pen.n) * 0.001   #for each pen
-    
-    env_Umin_cons = pen.U - fish.U_min
-    env_Umax_cons = fish.U_max - pen.U
-    env_tempmin_cons = pen.temp - fish.temp_min
-    env_tempmax_cons = fish.temp_max - pen.temp
-    env_salinitymin_cons = pen.salinity - fish.salinity_min
-    env_salinitymax_cons = fish.salinity_max - pen.salinity
-    env_O2_min_cons = pen.O2_in - fish.O2_min
-    
-    #print('cons:', P_gen_cons, fish_yield_cons)
+    P_gen_cons = (wec.annual_energy - pen.power) / wec.annual_energy
 
+    fish_yield_cons = (carrying_capacity - fish_yield) / carrying_capacity
+ 
+    pen_ratio_low_cons = (pen.D - pen.H) / pen.D
```

Comment:
Consider explaining what these ratios are for
```suggestion
    pen_ratio_low_cons = (pen.D - pen.H) / pen.D
```

Reply:
added explanation as a comment to code